### PR TITLE
Touch-up some README files

### PR DESCRIPTION
**Pull request by eloots**
- Fix a typo
- Minor rewording

### CODE REVIEW

File: alpakka-sample-http-csv-to-kafka-scala/step_003_parse_csv/README.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 ### Description
 
-The binary data in @scaladoc:[ByteString](akka.util.ByteString)s is passed into @extref:[Alpakka CSV](alpakka-docs:data-transformations/csv.html) to be parsed and per line converted into Maps. The stream elements become `Map[String, ByteString]`, one per line in the CSV file using the file's headers as keys.
+The binary data in @scaladoc:[ByteString](akka.util.ByteString)s is passed into @extref:[Alpakka CSV](alpakka-docs:data-transformations/csv.html) to be parsed and converted per line into Maps. The stream elements becomes a `Map[String, ByteString]`, one key/value pair per line in the CSV file, using the file's field headers as keys.
```
Comment (ennru) [inline alpakka-sample-http-csv-to-kafka-scala/step_003_parse_csv/README.md]: It is not "one key/value pair per line" it's one pair per column.

File: alpakka-sample-http-csv-to-kafka-scala/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.idea/
```
Comment (raboof) [inline alpakka-sample-http-csv-to-kafka-scala/.gitignore]: :+1: to add this here, but for convenience you might want to add it to your [global gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore)

File: alpakka-sample-http-csv-to-kafka-scala/step_003_parse_csv/README.md (hunk 1 above)
Comment (eloots) [reply alpakka-sample-http-csv-to-kafka-scala/step_003_parse_csv/README.md]: Correct, my bad - Reformulating as the `one per line in the CSV file using the file's headers as keys` was a bit confusing to me.
Will amend the PR.

File: alpakka-sample-http-csv-to-kafka-scala/.gitignore (hunk 2 above)
Comment (eloots) [reply alpakka-sample-http-csv-to-kafka-scala/.gitignore]: @raboof Thanks for the suggestion! Do the akka projects not add `.idea` in the project's `.gitignore`? In other words, do they count on the users having `.idea` in their global `.gitignore`?

**Pull request merged**